policy/policy-authority-data_test.go

```diff
@@ -53,6 +56,32 @@ func TestLoadAndDumpRules(t *testing.T) {
 	test.AssertEquals(t, dumped.Blacklist[0], load.Blacklist[0])
 }
 
+// An implementation of the gorpDbMap interface that always returns an error
+// from SelectOne.
+type failureDB struct{}
```

Comment:
We only want to use `mocks` for mocks that are shared across packages. This particular mock is only needed locally. We could in theory have use for a more general failureDB struct, but that would mean making it much broader than we need here in order to satisfy all the necessary interfaces.